Wireframe/index.html

```diff
@@ -1,33 +1,53 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200..700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="styles.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wireframe</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>**Key Concepts in Software Development: README, Wireframes, and Git Branches**</h1>
+      <p>
+        This article explains three fundamental concepts in software development: the purpose of a README file for project documentation, the role of wireframes in UI/UX design, and the function of branches in Git for managing code versions and collaboration.
+      </p>
+    </header>
+    <main>
+      <article>
+        <img src="wireframe.jpg" alt= "wireframe">
+        <h2>What is the purpose of a wireframe?</h2>
+        <p>
+          A wireframe is a low-fidelity visual representation of a webpage or app layout. It outlines structure, navigation, and content placement without detailed design elements. Wireframes help designers and stakeholders plan functionality, user experience, and flow, ensuring alignment before development begins, and serving as a blueprint for the final product.
+        </p>
+        <a href="https://adamfard.com/blog/wireframes#:~:text=Wireframes%20are%20a%20visual%20guide,a%20UI%20has%20a%20purpose.">Read more</a>

```

Comment:
Gosh, these urls are very long aren't they? Where did this text on the end come from? What is it doing for us in our code?